frappe/utils/data.py

```diff
@@ -22,6 +22,15 @@
 DATETIME_FORMAT = DATE_FORMAT + " " + TIME_FORMAT
 
 # datetime functions
+def get_string_date(date_object=None):
+	"""
+	Converts datetime.date object to string date (yyyy-mm-dd)
+	"""
+	if not date_object:
+		return frappe.utils.nowdate()
+	else:
```

Comment:
can you put this in the bottom? Also are you using it somewhere? Otherwise there will be lots of unused / undocumented functions in `frappe.utils`

Also the function should also accept a string date